PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingViewController.swift

```diff
@@ -47,28 +47,47 @@ class PetListingViewController: UIViewController
         self.collectionView.dataSource = self
         self.collectionView.alwaysBounceVertical = true
         self.collectionView.backgroundColor = UIColor.groupTableViewBackgroundColor()
-        self.collectionView.collectionViewLayout = CustomHomeCollectionViewFlowLayout()
+        self.collectionView.collectionViewLayout = HomePortraitCollectionViewLayout()
         
         refreshControl.addTarget(self, action: #selector(refreshTriggered), forControlEvents: .ValueChanged)
         self.collectionView.addSubview(refreshControl)
         
         loadPets()
-
     }
 
+    ////////////////////////////////////////////////////////////
+
     override func viewWillAppear(animated: Bool)
     {
         super.viewWillAppear(animated)
         self.navigationItem.title = NSLocalizedString("Town Of Lady Lake", comment: "")
     }
 
+    ////////////////////////////////////////////////////////////
```

Comment:
I dunno, this is the way I code.  12 years of coding standards at my last job.  In lieu of a coding standard of our own, and given that at one time, no one else was developing on this project, I adopted my personal coding standard.  If my coding style really bothers everyone, I suppose I can change it...